src/modules/functions.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable import/no-cycle */
 /* eslint-disable no-plusplus */
-import { tasks } from '../index.js';
+import tasks from '../index.js';
 
 // eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] As mentioned by the previous reviewer, Kindly make sure you fix all linters errors instead of disabling them.